target/linux/ipq40xx/config-4.19

```diff
@@ -283,6 +284,14 @@ CONFIG_MFD_SYSCON=y
 CONFIG_MIGHT_HAVE_CACHE_L2X0=y
 CONFIG_MIGHT_HAVE_PCI=y
 CONFIG_MIGRATION=y
+CONFIG_MMC=y
+CONFIG_MMC_BLOCK=y
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_IO_ACCESSORS=y
+CONFIG_MMC_SDHCI_MSM=y
+# CONFIG_MMC_SDHCI_PCI is not set
+CONFIG_MMC_SDHCI_PLTFM=y
+# CONFIG_MMC_TIFM_SD is not set
```

Comment:
Is it possible to build the SDHCI as a module instead?

Reply:
Making this built-in will cause unnecessary kernel bloat.
You can make a separate package just like `kmod-sdhci-mt7620`

Reply:
Do we really need another kmod?
There is no practical benefit to it, ipq40xx is not resource-constrained and few kilobytes are not an issue here.
MT7620 is much more space-constrained usually.